pkg/services/sqlstore/migrations/ualert/permissions.go

```diff
@@ -279,3 +287,18 @@ func (m *migration) getACL(orgID, dashboardID int64) ([]*dashboardAcl, error) {
 	err = m.sess.SQL(rawSQL, orgID, dashboardID).Find(&result)
 	return result, err
 }
+
+// getOrgsThatHaveFolders returns a list of orgID that have at least one folder
```

Comment:
This method returns a map (or "set"), not a list.

Reply:
a list, as well as a collection, is a more general description of a map or set.